src/components/Posts/PostItem.tsx

```diff
@@ -153,6 +164,13 @@ const PostItem = ({ post }: PostProps) => {
         </Stack>
       </CardActions>
       <Divider light />
+      {openModal && (
```

Comment:
Similar story here to the comment I made in `ExpandedPostItem.tsx`, I suggest we choose either the parent or child to control the render, not both